xlsx/tests/test.rs

```diff
@@ -334,3 +334,68 @@ fn no_export() {
     }
     fs::remove_dir_all(&dir).unwrap();
 }
+
+// This test verifies whether exporting the merged cells functionality is happening properly or not.
+// It first loads the excell having the merged cell and exports it to another xlsx and verifies whether merged
+// cell node is same in both of the xlsx file or not.
+#[test]
+fn test_exporting_merged_cells() {
+    let temp_file_name = "temp_file_test_export_merged_cells.xlsx";
+    let expected_merge_cell_ref = {
+        // loading the xlsx file containing merged cells
+        let example_file_name = "tests/example.xlsx";
+        let mut model = load_from_xlsx(example_file_name, "en", "UTC").unwrap();
+        let expected_merge_cell_ref = model
+            .workbook
+            .worksheets
+            .first()
+            .unwrap()
+            .merge_cells
+            .clone();
+        // exporting and saving it in another xlsx
+        model.evaluate();
+        save_to_xlsx(&model, temp_file_name).unwrap();
+        expected_merge_cell_ref
+    };
+    {
+        let mut temp_model = load_from_xlsx(temp_file_name, "en", "UTC").unwrap();
+        {
+            // loading the previous file back and verifying whether
+            // merged cells got exported properly or not
+            let got_merge_cell_ref = &temp_model
+                .workbook
+                .worksheets
+                .first()
+                .unwrap()
+                .merge_cells
+                .clone();
+            assert_eq!(expected_merge_cell_ref, *got_merge_cell_ref);
+            fs::remove_file(temp_file_name).unwrap();
+        }
+        {
+            // this block is to verify that if there are no
+            // merged cells, exported xml should not have the
+            // <mergeCells/> xml node
+            temp_model
```

Comment:
This is fine, but I think we probably should use an (not yet existing) API  call for this

Reply:
ohh okay. Let me know if its necessary, I can  implement get, set and clear functions for merged cells in worksheet

Reply:
I have added a commit which has APIs for get and clear on merge cells. if not needed, will revert the last commit.

Reply:
Hi @varuntumbe, no no need to do this just yet. I think it is fine if we use the json structure for the purposes of this PR.
Let's deal with the correct API for merge cell on a separate commit.  Thanks!

Reply:
Got it Thanks @nhatcher